mne/io/kit/kit.py

```diff
@@ -910,8 +833,13 @@ def read_raw_kit(input_fname, mrk=None, elp=None, hsp=None, stim='>',
 
     Notes
     -----
-    If mrk, hsp or elp are array_like inputs, then the numbers in xyz
-    coordinates should be in units of meters.
+    ``elp`` and ``hsp`` are usually the exported text files (*.txt) from the
```

Comment:
```suggestion
    ``elp`` and ``hsp`` are usually the exported text files (\*.txt) from the
```

Reply:
Uhhh I was looking for an issue with the double backticks.. thanks!